packtools/sps/validation/article_xref.py

```diff
@@ -60,7 +60,9 @@ def validate_xref_rid_has_corresponding_element_id(self, error_level="ERROR"):
                     is_valid=is_valid,
                     expected=rid,
                     obtained=rid if is_valid else None,
-                    advice=f'Check if xref[@rid="{rid}"] is correct or insert the missing {element_name}[@id="{rid}"]',
+                    advice=f'mark the information correctly in: <xref rid="{rid}" />. '
+                           f'No matching <{element_name} id="{rid}" /> found. '
+                           f'Fix <xref rid="{rid}" /> or insert <{element_name} id="{rid}" />.',
```

Comment:
@Rossi-Luciano 'mark the information correctly in' é muito vago.

```python
element_name = xref.get("element_name")
ref_type = xref.get("ref-type")

```
`Found <xref rid="{rid}" ref-type="{ref_type}">...</xref>, but not found the corresponding <{element_name} id="{rid}">` Check if the value rid="" and ref-type="" are correct and check if <{element_name}> have correct id=""`

Reply:
OK @robertatakenaka